courses/machine_learning/deepdive2/supplemental/solutions/question_answering_rnn.ipynb

```diff
@@ -0,0 +1,684 @@
+{
```

Comment:
<ul><li>Break lines according to <a href="https://www.python.org/dev/peps/pep-0008/#maximum-line-length" target="_blank">pep8</a> (no more than 80 chars per line), or, in this case import "pad_sequence" already in the import section</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1325/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>